packages/supabase/src/index.ts

```diff
@@ -62,13 +62,15 @@ const handleError = (error: PostgrestError) => {
 
 const dataProvider = (supabaseClient: SupabaseClient): DataProvider => {
     return {
-        getList: async ({ resource, pagination, filters, sort }) => {
+        getList: async ({ resource, pagination, filters, sort, metaData }) => {
             const current = pagination?.current || 1;
             const pageSize = pagination?.pageSize || 10;
 
             const query = supabaseClient
                 .from(resource)
-                .select("*", { count: "exact" })
+                .select(metaData?.select ? metaData?.select : "*", {
+                    count: "exact",
+                })
```

Comment:
```suggestion
                .select(metaData?.select ?? "*", {
                    count: "exact",
                })
```
How would you like it to be this way?

Reply:
Much better, thanks!